pkg/services/folder/folderimpl/folder.go

```diff
@@ -947,13 +942,41 @@ func (s *Service) Move(ctx context.Context, cmd *folder.MoveFolderCommand) (*fol
 			return folder.ErrInternal.Errorf("failed to move legacy folder: %w", err)
 		}
 
+		if err := s.publishFolderFullPathUpdatedEvent(ctx, f.Updated, cmd.OrgID, cmd.UID); err != nil {
+			return err
+		}
+
 		return nil
 	}); err != nil {
 		return nil, err
 	}
 	return f, nil
 }
 
+func (s *Service) publishFolderFullPathUpdatedEvent(ctx context.Context, timestamp time.Time, orgID int64, folderUID string) error {
+	descFolders, err := s.store.GetDescendants(ctx, orgID, folderUID)
```

Comment:
not blocking for this but it would be nice to have some [tracing](https://github.com/grafana/grafana/blob/main/contribute/backend/instrumentation.md#traces0

Reply:
Added tracing: https://github.com/grafana/grafana/pull/88376/commits/dc0da4178c18e8ca7eb2b4673bcaa00eef6429c8